cmd/lncli/commands.go

```diff
@@ -33,12 +33,13 @@ import (
 
 // TODO(roasbeef): expose all fee conf targets
 
-const defaultRecoveryWindow int32 = 2500
-
 const (
-	defaultUtxoMinConf = 1
+	defaultRecoveryWindow int32 = 2500
```

Comment:
This change causes the linter to fail.

Reply:
yeah and turns out `defaultUtxoMinConf` needs to be untyped. reverted